src/stepper/Step.js

```diff
@@ -83,7 +83,7 @@ Step.propTypes = {
   /**
    * The step number.
    */
-  step: PropTypes.string,
+  step: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
I think this should only be number, happy to chat about it

Reply:
@tlouth19 what if we wanted a step A, B, C ?

Reply:
I don't think we would ever do that

Reply:
Well what if we want Step 🍌  ... Step 🦧  ...  Step 🌵  ?